examples/scripts/iframe/index.mjs

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import Babel from '@babel/standalone';
+import Handlebars from 'handlebars';
+import formatters from '../../src/app/helpers/formatters.mjs';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const MAIN_DIR = `${__dirname}/../../`;
+
+const EXAMPLE_CONSTS = [
+    "vshader",
+    "fshader",
+    "fshaderFeedback",
+    "fshaderCloud",
+    "vshaderFeedback",
+    "vshaderCloud"
+];
+
```

Comment:
Even in pure JS its possible to come quite far with type safety:
```suggestion
/**
 * @param {string} data 
 * @param {string} name 
 */
```

Reply:
I'd like to make a pass of the whole examples project typings. That'll be in a later PR though.